pkg/slo-controller/noderesource/resource_calculator.go

```diff
@@ -120,7 +104,8 @@ func (r *NodeResourceReconciler) updateNodeResource(node *corev1.Node, nr *frame
 func (r *NodeResourceReconciler) updateNodeExtensions(node *corev1.Node, nodeMetric *slov1alpha1.NodeMetric, podList *corev1.PodList) error {
 	// update device resources
 	if err := r.updateDeviceResources(node); err != nil {
-		klog.Errorf("failed to update device resources for node %s, err: %v", node.Name, err)
+		klog.V(4).InfoS("failed to update device resources for node", "node", node.Name,
```

Comment:
why use InfoS here?

Reply:
The `Reconcile` will log the error of `updateNodeExtensions`, so here it just log for `updateDeviceResources` with a higher verbosity.